js-intro.js

```diff
@@ -12,32 +12,61 @@ var temp2 = 350
 var temp3 = 212
 
 
+// create a function that takes in one argument
+// tells if the argument is above, below, or at 212
+const tempTest = (temp) => {
+    if (temp > 212) {
+        return `${temp} is above the boiling point.`
+    } else if (temp < 212) {
+        return `${temp} is below the boiling point.`
+    } else if (temp === 212) {
```

Comment:
Great job using `else if` rather than `else` to communicate your intention with your code.